homeassistant/components/unifi/config_flow.py

```diff
@@ -108,12 +110,10 @@ async def async_step_user(
                 errors["base"] = "service_unavailable"
 
             else:
-                self.site_ids = {site["_id"]: site["name"] for site in sites.values()}
-                self.site_names = {site["_id"]: site["desc"] for site in sites.values()}
-
                 if (
                     self.reauth_config_entry
-                    and self.reauth_config_entry.unique_id in self.site_names
+                    and self.reauth_config_entry.unique_id is not None
+                    and self.reauth_config_entry.unique_id in self.sites
```

Comment:
how does this check work if `self.sites` is a `Sites` instance and `unique_id` is a `str`?

Reply:
Excellent question @raman325 ! The base class ApiHandler has a contains dunder method that qualifies existence of the string in the class.
https://github.com/Kane610/aiounifi/blob/4a8bb60b522c3bc7d872f27da34c405a7a298652/aiounifi/interfaces/api_handlers.py#L158

Reply:
Sweet! Was hoping that was the case, clever!

Reply:
Sometimes I succeed 🤣